linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev/rule.yml

```diff
@@ -24,7 +24,7 @@ identifiers:
 references:
     cis@rhel8: 1.1.5
     cis@ubuntu1804: 1.1.14
-    stigid@ol7: OL07-00-021022
+    stigid@ol7: OL07-00-021024
```

Comment:
Duplicated STIGIDs pose a bit of a problem. when results are translated into STIG viewer format, for example, only one of the rules will have the results incorporated. Also, `stig_overlay.xml` will have one of these three rules. I know that DISA has deliberately merged these rules into one single requirement, but that's the opposite we ultimately want to achieve. We want to have a single configuration item per rule. I have tried to merge these three rules together in the past but with no success, see the discussion here: https://github.com/ComplianceAsCode/content/pull/6087

I'm not complaining about this change itself, but more about explaining the context behind the problems that people might have with STIG results and mapping.

Reply:
Got it. I'll follow the same approach and leave the old ids.